services/order.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+
+from db.models import Order
+from django.db import transaction
+from typing import List
+from db.models import MovieSession, Ticket
+from db.models import User
+
+
+@transaction.atomic
+def create_order(
+        tickets: List[dict],
+        username: str,
+        date: datetime = None
+) -> None:
+    user, created = User.objects.get_or_create(username=username)
+    order, created = Order.objects.get_or_create(user=user)
```

Comment:
```suggestion
    order = Order.objects.create(user)
```
One use can have many orders